docker-entrypoint.sh

```diff
@@ -15,18 +19,38 @@ if [ ! -z ${certfile} ] && [ "${certfile}" != "null" ] ; then
             -new $(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 20)
 fi
 
-for COMMAND in $COMMANDS
+for comarg in $COMMANDS
 do
-    if [ "${COMMAND}" = "run" ] ; then
-        FULL_COMMAND="exec java --illegal-access=deny -Drems.config=config/config.edn -jar rems.jar"
+    # Check if handling a command with parameters
+    if [[ "${comarg}" == \(* ]] ; then
+        parameters=true
+        cmd="${comarg#"("}"
+        continue
+    # Handle parameter instead of command
+    elif [ "${parameters}" = true ] ; then
+        cmd+=" ${comarg%")"}"
+        if [[ "${comarg}" == *\) ]] ; then
+        parameters=false
+        else
+            continue
+        fi
+    # Handle run command
+    elif [ "${comarg}" = "run" ] ; then
+        cmd_prefix="exec"
+        cmd=""
+    # All other commands
```

Comment:
I commend your effort of writing a parser in bash but I'd rather not have one :sweat_smile: 

How about introducing a new environment variable (for example `CMDS`) that would contain a ;-separated list of commands with arguments?

Reply:
Sure. I thought about that, but I came to a conclusion that being able to use the same parameter would be simpler from the user's perspective. Ill try to come up with a more descriptive name than `CMDS` though.

Reply:
I don't think our docker images have that many users yet, so we might even be able to drop the old `COMMANDS` after a release or two